WordPress/Classes/ViewRelated/NUX/LoginEmailViewController.swift

```diff
@@ -1,35 +1,328 @@
 import UIKit
 
-protocol LoginViewController {
-    func setupNavBarIcon()
-}
+/// This is the first screen following the log in prologue screen if the user chooses to log in.
+///
+class LoginEmailViewController: NUXAbstractViewController, SigninKeyboardResponder, LoginViewController {
+    @IBOutlet var instructionLabel: UILabel!
```

Comment:
Outlets need to be declared as an optional type since they are composed via the storyboard or a xib rather than assigned during initialization. Defining them as implicitly unwrapped is recommended by Apple:
`When you declare an outlet in Swift, you should make the type of the outlet an implicitly unwrapped optional`
https://developer.apple.com/library/content/documentation/Swift/Conceptual/BuildingCocoaApps/WritingSwiftClassesWithObjective-CBehavior.html